src/v2/Components/ZendeskWrapper.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+import Zendesk from "react-zendesk"
+import { data as sd } from "sharify"
+
+export const ZendeskWrapper = () => {
+  return <Zendesk zendeskKey={sd.ZENDESK_KEY} />
```

Comment:
We can conditionally render the Zendesk widget based on the presence of the `sd.ZENDESK_KEY` so we can control the rendering by setting/unsetting the environment variable.